distributed/protocol/tests/test_serialize.py

```diff
@@ -445,21 +445,6 @@ def dumps(f):
     assert "Hello-123" in str(info.value)
 
 
-@gen_test()
-async def test_profile_nested_sizeof():
-    # https://github.com/dask/distributed/issues/1674
-    n = 500
-    original = outer = {}
-    inner = {}
-
-    for _ in range(n):
-        outer["children"] = inner
-        outer, inner = inner, {}
-
-    msg = {"data": original}
-    frames = await to_frames(msg)
```

Comment:
why remove this?

Reply:
I guess I could reintroduce it, with a change in name, with `n = 400` to test that the maximum recursion depth of sizeof doesn't accidentally halve in the future?